sstables/sstables_manager.cc

```diff
@@ -186,9 +186,15 @@ size_t sstables_manager::get_memory_available_for_reclaimable_components() const
 }
 
 future<> sstables_manager::components_reclaim_reload_fiber() {
+    auto _components_memory_reclaim_threshold_observer = _db_config.components_memory_reclaim_threshold.observe([&] (const double&) {
```

Comment:
We only name members with `_` prefix.

Reply:
Taking `double` by reference is not worth it, I suggest to use `double` unless the `observe` demands `const T&`.